.github/workflows/integration-tests.yml

```diff
@@ -52,7 +57,7 @@ jobs:
         key: [8, 16]
     env:
       TARGET: "x86_64-unknown-linux-musl"
-      BUILD_MODE: "release"
+      BUILD_MODE: "debug"
```

Comment:
Revert this change in this request. This change is already made in https://github.com/qoollo/bob/pull/524